packages/dev/core/src/Compat/index.ts

```diff
@@ -1 +1 @@
-export { CompatibilityOptions } from "./compatibilityOptions";
+export * from "./compatibilityOptions";

```

Comment:
I had to make this change or I get this error from running the sandbox locally:
![image](https://github.com/user-attachments/assets/fa634a63-8e64-441e-b872-ec254de684c2)